content/docs/studio/get-started.md

```diff
@@ -59,9 +59,8 @@ details]).
 
    ![](https://static.iterative.ai/img/studio/view_components.png)
 
-6. You can [train on the cloud and submit new experiments] by changing
-   hyperparameters and datasets, and you can run on your own cloud
-   infrastructure.
+6. You can [submit new experiments] by changing hyperparameters and datasets,
+   and you can run the experiments on your own cloud infrastructure.
```

Comment:
Why change this? I think I added this pretty recently to stress that it's not only about changing hyperparameters and running experiments. You could use this workflow for any training on the cloud.

Reply:
This one is just a rewording. I couldn't understand `and you can run on your own cloud infrastructure`. It probably means `and you can run (the training?) on your own cloud infrastructure`. Does this sentence convey your message `You can train on the cloud, including on your own cloud infrastructure, and submit new experiments by changing hyperparameters and datasets.`?

Reply:
Yes, sounds good, @tapadipti

Reply:
Done [here](https://github.com/iterative/dvc.org/pull/4638/commits/8116b516fad73912cf266d3121e19f46b9c63846)